docs/Node Operators/running-a-chainlink-node.md

```diff
@@ -16,10 +16,41 @@ It's important to note that nodes can fulfill requests for open APIs out-of-the-
 
 If you would like to provide data from an authenticated API, you can add an [external adapter](../external-adapters/) to enable connectivity through the Chainlink node.
 
-Hardware requirements are light. The only heavy part is you'll need a blockchain node connection. If you use a 3rd party (defined below), you can use a machine with as little as 10GB of storage and 4GB of RAM.
-
 ![Chainlink Node Diagram](/files/ab5762f-end-to-end-diagram.png)
 
+# Hardware Requirements
+
+## Chainlink Node
+
+Your Chainlink node should be run on a server that has a public IP address.
+
+### Minimum
+
+To get started running a Chainlink node, you will need a machine with at least **2 cores** and **4 GB of RAM**. 
+
+### Recommended 
+
+The requirements for running a Chainlink node scale as the number of jobs your node services also scales.  For nodes with over 100 jobs, you will need at least **4 cores** and **8GB of RAM**.  
+
+
+## PostgreSQL Database
+
+In addition to running a Chainlink node, you will also need a PostgreSQL database.  Please use a version >= 11, and be sure that your DB host provides access to logs.
+
+### Minimum
+
+The minimum requirements for the database are **2 cores**, **4GB of RAM**, and **100 GB of storage**.
+
+### Recommended
+
+Similar to the Chainlink node, requirements increase as you service more jobs.  For more than 100 jobs, your database server will need at least **4 cores**, **16 GB of RAM**, and **100 GB of storage**. 
+
+If using AWS, use dedicated IOPS (not the burstable one).
```

Comment:
```suggestion
If you run your node on AWS, use an instance type with dedicated core time. [Burstable Performance Instances](https://aws.amazon.com/ec2/instance-types/#Burstable_Performance_Instances) have a limited number of [CPU credits](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/burstable-credits-baseline-concepts.html).
```

Reply:
I like your improvement here.  Is it possible to make the second sentence a bit more explicit?  "Burstable Performance instances have...and so should not be used"

Reply:
Made an edit to the suggestion. Is that more clear?